cumulus/parachains/runtimes/assets/asset-hub-rococo/tests/tests.rs

```diff
@@ -1355,3 +1356,17 @@ fn change_xcm_bridge_hub_ethereum_base_fee_by_governance_works() {
 		},
 	)
 }
+
+#[test]
+fn location_conversion_works() {
+	let alice_on_bh = Location::new(
+		1,
+		[
+			Parachain(1111),
+			Junction::AccountId32 { network: None, id: AccountId::from(ALICE).into() },
+		],
+	);
```

Comment:
`alice_on_bh` implies BH para id will be used, so either the right para id (not 1111) or change name to `sibling_alice` or something similar.

Reply:
fixed